make/autoconf/platform.m4

```diff
@@ -666,14 +666,14 @@ AC_DEFUN([PLATFORM_CHECK_DEPRECATION],
 [
   AC_ARG_ENABLE(deprecated-ports, [AS_HELP_STRING([--enable-deprecated-ports@<:@=yes/no@:>@],
       [Suppress the error when configuring for a deprecated port @<:@no@:>@])])
-  # if test "x$OPENJDK_TARGET_CPU" = xx86; then
-  #   if test "x$enable_deprecated_ports" = "xyes"; then
-  #     AC_MSG_WARN([The x86 port is deprecated and may be removed in a future release.])
-  #   else
-  #     AC_MSG_ERROR(m4_normalize([The 32-bit x86 port is deprecated and may be removed in a future release.
-  #       Use --enable-deprecated-ports=yes to suppress this error.]))
-  #   fi
-  # fi
+  if test "x$OPENJDK_TARGET_CPU" = xx86; then
+    if test "x$enable_deprecated_ports" = "xyes"; then
+      AC_MSG_WARN([The x86 port is deprecated and may be removed in a future release.])
+    else
+      AC_MSG_ERROR(m4_normalize([The 32-bit x86 port is deprecated and may be removed in a future release.
```

Comment:
Shouldn't this wording be the same in both cases?

Reply:
Right, it should. The only difference is whether this message is an error or a warning. Fixed in new commit.